ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradePlannerWindow.xaml

```diff
@@ -192,6 +195,218 @@
+										Grid.Column="0"
+										Grid.ColumnSpan="2"
+										>
+										<Grid.RowDefinitions>
+											<RowDefinition Height="Auto" />
+											<RowDefinition Height="Auto" />
+											<RowDefinition Height="Auto" />
+											<RowDefinition Height="*" />
+										</Grid.RowDefinitions>
+
+										<Grid.ColumnDefinitions>
+											<ColumnDefinition Width="Auto" />
+											<ColumnDefinition Width="Auto" />
+											<ColumnDefinition Width="Auto" />
+											<ColumnDefinition Width="Auto" />
+										</Grid.ColumnDefinitions>
+
+										<common1:ImageTextControl
+											Grid.Row="0"
+											Grid.Column="0"
+											Margin="8 0"
+											HorizontalAlignment="Left"
+											Image="{Binding Source={x:Static resource:IconContent.ResourceFuel}, Converter={StaticResource ImageSourceConverter}}"
+											Text="{Binding Cost.Fuel}"
+											/>
+
+										<common1:ImageTextControl
+											Grid.Row="0"
+											Grid.Column="1"
+											Margin="8 0"
+											HorizontalAlignment="Left"
+											Image="{Binding Source={x:Static resource:IconContent.ResourceAmmo}, Converter={StaticResource ImageSourceConverter}}"
+											Text="{Binding Cost.Ammo}"
+											/>
+
+										<common1:ImageTextControl
+											Grid.Row="0"
+											Grid.Column="2"
+											Margin="8 0"
+											HorizontalAlignment="Left"
+											Image="{Binding Source={x:Static resource:IconContent.ResourceSteel}, Converter={StaticResource ImageSourceConverter}}"
+											Text="{Binding Cost.Steel}"
+											/>
+
+										<common1:ImageTextControl
+											Grid.Row="0"
+											Grid.Column="3"
+											Margin="8 0"
+											HorizontalAlignment="Left"
+											Image="{Binding Source={x:Static resource:IconContent.ResourceBauxite}, Converter={StaticResource ImageSourceConverter}}"
+											Text="{Binding Cost.Bauxite}"
+											/>
+
+
+										<common1:ImageTextControl
+											Grid.Row="1"
+											Grid.Column="0"
+											Margin="8 0"
+											HorizontalAlignment="Left"
+											Image="{Binding Source={x:Static resource:IconContent.ItemDevelopmentMaterial}, Converter={StaticResource ImageSourceConverter}}"
+											Text="{Binding Cost.DevelopmentMaterial}"
+											/>
+
+										<common1:ImageTextControl
+											Grid.Row="1"
+											Grid.Column="2"
+											Margin="8 0"
+											HorizontalAlignment="Left"
+											Image="{Binding Source={x:Static resource:IconContent.ItemModdingMaterial}, Converter={StaticResource ImageSourceConverter}}"
+											Text="{Binding Cost.ImprovmentMaterial}"
+											/>
+
+										<ItemsControl
+											Grid.Row="2"
+											Grid.Column="0"
+											Grid.ColumnSpan="4"
+											ItemsSource="{Binding Cost.RequiredEquipments}"
+											>
+											<ItemsControl.ItemTemplate>
+												<DataTemplate>
+													<StackPanel Margin="4 4 0 0" Orientation="Horizontal">
```

Comment:
Add `Background="Transparent"` here.
Without the background, you need to hover over the text exactly to see the tooltip, which can look a bit confusing.